ext/web/02_timers.js

```diff
@@ -286,7 +287,7 @@ function runAfterTimeout(task, millis, timerInfo) {
       while (currentEntry !== null) {
```

Comment:
Would using a priority queue be a better idea than looping over a doubly linked list, especially in the cases where the user creates a large number of simultaneously running timers?

Reply:
@milaz we want to completely overhaul timers implementation so they work nicely with eg. "AsyncLocalStorage". I would prefer to punt on changing the underlying machinery until that happens, but I will definitely take your suggestion into account 👍